src/modules/esl-scrollbar/core/esl-scrollbar.ts

```diff
@@ -105,7 +105,7 @@ export class ESLScrollbar extends ESLBaseElement {
   protected bindTargetEvents(): void {
     if (!this.$target) return;
     if (document.documentElement === this.$target) {
-      this.$$on(this._onScrollOrResize);
+      this.$$on({event: 'scroll', target: window}, this._onScrollOrResize);
```

Comment:
```suggestion
      this.$$on({event: ''resize scroll'', target: window}, this._onScrollOrResize);
```
According to initial metainformation on this._onScrollOrResize